docs/content/config/advanced/mail-forwarding/gmail-smtp.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Mail Forwarding | GMAIL SMTP'
+---
+
+[GMAIL SMTP (Simple Message Transport Protocol)][gmail-smtp] provides a simple way for cloud based applications to send and receive email.
+
+!!! example "Configuration via ENV"
+
+    [Configure a relay host in DMS][docs::relay] to forward all your mail through GMAIL SMTP:
+
+    - `RELAY_HOST` should match your [SMTP Server Endpoint][gmail-smtp::relay-host].
+    - `RELAY_PORT` should be set to [one of the supported Gmail SMTP ports][gmail-smtp::relay-port] (_eg: 587 for STARTTLS_).
+    - `RELAY_USER` and `RELAY_PASSWORD` should be set to your [Gmail Account ID][gmail-smtp::account-id].
+
+    ```env
+    RELAY_HOST=smtp.gmail.com
+    RELAY_PORT=587
+    # Alternative to RELAY_HOST + RELAY_PORT which is compatible with LDAP:
+    DEFAULT_RELAY_HOST=[smtp.gmail.com]:587
+
+    RELAY_USER=someone@gmail.com
+    RELAY_PASSWORD=xxx
+    ```
+
+!!! warning "Process of providing RELAY_PASSWORD"
+
+    You should use your [2-step verification app password][gmail-smtp::2-step-password], **not** your gmail account password.
+    `setup relay add-auth` is a better alternative, which manages the credentials via a config file.
+    
+!!! tip
+
+    If you have set up GMAIL SMTP, you can Filter messages for spam and viruses before they reach external recipients
+    and also apply email security and advanced Gmail settings to outgoing messages.
```

Comment:
This tip should clarify with a link if possible to Gmail configuration advice/feature?

Reply:
@polarathene It's in https://support.google.com/a/answer/2956491     first Paragraph!

Reply:
Oh thanks, missed that when looking through the links.

Is this a tip you found valuable? There isn't much guidance for more information, it seems more to just be for awareness so I'm not sure helpful it is to highlight from that resource verbatim.

Reply:
I agree with your opinion. i'll remove it.